public/js/demo.js

```diff
@@ -0,0 +1,238 @@
+/**
```

Comment:
That might be something to discuss with Derk: what is the proper name for this file? `demo.js` does not help to know what it does, or what it is used for. Can it be renamed?